src/loaders/ncbi_downloader/ncbi_downloader.py

```diff
@@ -1,74 +1,76 @@
 """
-PROTOTYPE - Download genome files from NCBI FTP server.
-
-usage: ncbi_downloader.py [-h] --download_file_ext DOWNLOAD_FILE_EXT [DOWNLOAD_FILE_EXT ...] --release_ver
-                          RELEASE_VER [--root_dir ROOT_DIR] [--source SOURCE] [--threads THREADS]
-                          [--overwrite]
+usage: ncbi_downloader.py [-h] --download_file_ext DOWNLOAD_FILE_EXT [DOWNLOAD_FILE_EXT ...] --release_ver {202,207,214,80,83,86,89,95}
```

Comment:
We talked about splitting this into two files:

* A helper module for downloading files from NCBI that just takes a list of IDs and downloads them to an output dir.
* A GTDB specific module that takes CLI arguments, downloads the GTDB specific files, parses them, and uses the helper module to download the NCBI files based on the IDs from the GTDB files.
* There's actually probably a 3rd file (or maybe this goes in loader_helpers) for creating the softlinks in `collectionssource`. That presumably could be reused by future NCBI related downloaders as well

This allows us to add more sources of IDs in the future and reuse the common NCBI related code.

That seems like it's going to change the PR a lot so it probably makes sense to do that and then I'll come back and review it in depth

Reply:
Updated!

Reply:
:+1: